modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -303,3 +303,42 @@ export class TextApiResponse {
         return await this.raw.text();
     };
 }
+
+/**
+ * Serializes an array's contents into the properties within the supplied formData instance.
+ * @param {FormData} formData - formData instance
+ * @param {string} paramName - name of the parameter (form value) being serialized
+ * @param {Array} arr - array of objects to be serialized
+ */
+export function serializeArrayObjectsToFormData(formData: FormData, paramName: string, arr: any[]) {
+    arr.forEach(elem => {
+        appendFormData(formData, elem, `${paramName}[]`)
+    });
+}
+
+/**
+ * Appends the data from an object to the supplied formData instance
+ * See: https://stackoverflow.com/a/43101878
+ * @param {FormData} formData - instance of FormData object
+ * @param {any} data - data (object?) to post to form data
+ */
+export function appendFormData(formData: FormData, data: any, previousKey?: string) {
+  if (data instanceof Object) {
+    Object.keys(data).forEach(key => {
+      const value = data[key];
+      if (value instanceof Object && !Array.isArray(value)) {
+        return appendFormData(formData, value, key);
+      }
+      if (previousKey) {
+        key = `${previousKey}[${key}]`;
```

Comment:
there are various ways to serialize arrays, using square brackets `[` `]` is one way. may be this should be configurable see e.g. https://github.com/OpenAPITools/openapi-generator/blob/13c94eeea05fb3b53d6442d1aef0ee9f7f5da007/modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache#L125

Reply:
Hey guys, this seems to be a minor issue. Any chance to get this fixed and the PR closed?

Reply:
I think we would welcome any updates to this if this feature has not been covered in other ways in the past 2 years. 

If you have the capacity, feel free to submit an updated PR! :) Happy to give you any pointers you may need.